mobile_verifier/src/reward_shares.rs

```diff
@@ -326,14 +325,16 @@ mod test {
         data_session,
         data_session::HotspotDataSession,
         heartbeats::HeartbeatReward,
-        speedtests::{Speedtest, SpeedtestAverages},
+        speedtests::Speedtest,
+        speedtests_average::{SpeedtestAverage, SpeedtestAverages},
```

Comment:
```suggestion
```
This line should be unnecessary since you `use super::*` to bring in those imports from the parent module already